### ISSUE TITLE
Section numbering bug

### ISSUE THREAD

**Issue by dgrigonis** [JelteF/PyLaTeX#156]
In [Line](https://github.com/JelteF/PyLaTeX/blob/master/pylatex/section.py#L35) change:

from
```
if self.numbering is not None:
    self.numbering = numbering
```
to 
```
if numbering is not None:
    self.numbering = numbering
```

**Comment by dgrigonis** [comment]
By the way, very nice package. Will be easy to contribute if I find anything worth implementing.

**Comment by JelteF** [comment]
This is fixed by #161
